musicrandomizer.py

```diff
@@ -37,6 +37,8 @@
     from mfvitools.insertmfvi import insertmfvi, byte_insert, int_insert, SampleIDError, FreeSpaceError
 
 JOHNNYDMAD_FREESPACE = ["53C5F-9FDFF", "310000-37FFFF", "410000-4FFFFF"]
+DEFAULT_PLAYLIST_FILE = 'default.txt'
```

Comment:
I don't think these are used anymore, but i didn't want to reorganize the usages of them in this file